src/utils/DragAndDrop.js

```diff
@@ -156,6 +156,18 @@ define(function (require, exports, module) {
             event = event.originalEvent || event;
 
             var files = event.dataTransfer.files;
+
+            var types = event.dataTransfer.types;
+            if(!files.length){// stop default behavior if a url is dragged in so the browser does not takeove
```

Comment:
Can `files` also be `null`? The file handler below seems to suggest that, so it might be worth guarding against it (`if(!files || !files.length)`.